### ISSUE TITLE
Not able to locate member from parent class, apparently

### ISSUE THREAD

**Issue by kinow** [Microsoft/pyright#14]
Hi, just created a `pyrightconfig.json` and started experimenting more with `pyright` against the project I am currently working on. It did find a couple of real bugs already :bug: thanks! But there are some that appear to be false positives.

One is this one where it complains about a missing member, which is defined in a parent class.

```
/home/kinow/Development/python/workspace/cylc/lib/cylc/loggingutil.py
  '_open' is not a known member of 'TimestampRotatingFileHandler' (141, 28)
```

However, the class [`TimestampRotatingFileHandler`](https://github.com/cylc/cylc/blob/611ea206722efe402c4446da04204cbe4b2f7119/lib/cylc/loggingutil.py#L66) extends [`logging.FileHandler`, which declares `_open`](https://github.com/python/cpython/blob/62be33870e2f8517314bf9c7275548e799296f7e/Lib/logging/__init__.py#L1178):

```python
    def _open(self):
        """
        Open the current base file with the (original) mode and encoding.
        Return the resulting stream.
        """
        return open(self.baseFilename, self.mode, encoding=self.encoding)
```

Cheers
Bruno

**Comment by kinow** [comment]
And just confirming, this is from the output of `pyright` ran as an external tool in PyCharm: 

![image](https://user-images.githubusercontent.com/304786/54893664-b6bc0f00-4f1b-11e9-9311-c8a321c2087c.png)

**Comment by kinow** [comment]
Oh, sorry, just realized that was a private method. I was reading it as `open`. Good warning! Closing.

**Comment by wearetvxq** [comment]
can you tell me how to install pyright in pycham, thanks

**Comment by erictraut** [comment]
Pyright isn't compatible with pycharm. It's a VS Code extension. It also runs on the command line.
